src/decoder_opus.cpp

```diff
@@ -29,6 +29,21 @@ static int custom_read(void *stream, unsigned char *ptr, int nbytes) {
 	return fread(ptr, 1, nbytes, f);
 }
 
+static int custom_seek(void *stream, opus_int64 offset, int whence) {
+	FILE* f = reinterpret_cast<FILE*>(stream);
+	return fseek(f, offset, whence);
+}
+
+static opus_int64 custom_tell(void *stream) {
+	FILE* f = reinterpret_cast<FILE*>(stream);
+	return ftell(f);
+}
+
+static int custom_close(void* stream) {
```

Comment:
minor style it: can you decide whether you use `void* var`, `void * var` and `void *var`?